.github/workflows/localstack.yml

```diff
@@ -0,0 +1,28 @@
+name: Run LocalStack tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  localstsack:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Start LocalStack
+        env:
+          LOCALSTACK_API_KEY: ${{ secrets.LOCALSTACK_API_KEY }}
+        run: |
+          # install LocalStack cli and awslocal
+          pip install localstack awscli-local[ver1]
+          # Make sure to pull the latest version of the image
+          docker pull localstack/localstack:0.14.0
```

Comment:
Ideally I think we'd put this in the `docker-compose.yml` so that all the docker images versions are controlled in one place.

Reply:
Yeah, definitely. This still isn't quite working on CI, so there's some more work to be done here for sure.